packages/utilities/psammead-storybook-helpers/src/text-variants.js

```diff
@@ -27,69 +30,79 @@ const TEXT_EXAMPLES = {
     script: 'arabic',
     dir: 'rtl',
     locale: 'ar',
+    path: '/arabic/articles/c1er5mjnznzo',
   },
   azeri: {
     text: 'Azərbaycanın siyasi ailələrinin ikinci evi – İngiltərə',
     longText:
       'Son aylarda müxtəlif səbəblərə görə Britaniya mətbuatında İngiltərədə yaşayan və ya burada biznesi olan azərbaycanlı məmurların ailə üzvlərinin adları qeyd olunub.',
     script: 'latinDiacritics',
     locale: 'az',
+    path: '/azeri/articles/c5k08pqnzexo',
   },
   bengali: {
     text: 'ভিসা ফুরিয়ে যাওয়ায় ক্রিকেটার সাইফের জরিমানা',
     longText:
       'বিমানবন্দরে যাওয়ার আগে সাইফ দেখেন, তার ভিসার মেয়াদ দুদিন আগেই শেষ হয়ে গেছে। তখন জরিমানা দিয়ে তাকে আবার এক্সিট পারমিট নিতে হয়।',
     script: 'bengali',
     locale: 'bn',
+    path: '/bengali/articles/c6p3yp5zzmeo',
   },
   burmese: {
     text: 'အောက်စဖို့ဒ် ဆရာတော် ပါမောက္ခ ဒေါက်တာအရှင်ဓမ္မသာမိ',
     longText:
       'တပ်မတော်နဲ့ ကရင်နယ်ခြားစောင့်တပ် BGF ပူးပေါင်းတပ်ဖွဲ့က MNLA ရဲ့ မြို့နယ်ခွဲရုံး ဖြစ်တဲ့ ဟင်္သာတိုင် ဂိတ်စခန်းနဲ့ ဂျပန်ရေတွင်းတို့ကို ထိန်းချုပ်ထားတယ်လို့ ဆိုပါတယ်။',
     script: 'burmese',
     locale: 'my',
+    path: '/burmese/articles/c3w1kwwmm5yo',
   },
   news: {
     text: 'Could a computer ever create better art than a human?',
     longText:
       'The critic, author, poet and TV host was known for his witty commentary on international television.',
     script: 'latin',
     locale: 'en',
+    path: '/news/articles/cn7k01xp8kxo',
```

Comment:
very nitpicky, but `news` is the only service that is not in alphabetical order. could you fix that pls 🙏

Reply:
😄I like the attention to detail here, thanks @thekp

Reply:
I've just been working on this package for a while now, just always forgot to update it myself 😅